### DISCUSSION TITLE
Welcome Screen edit

### DISCUSSION THREAD

**Discussion by ghost** [atom/atom#22262] [Q&A] [answered]
How can I change the way the welcome screen looks?

![image](https://user-images.githubusercontent.com/78956635/115819704-12b83100-a41d-11eb-8d18-496d8741a94a.png)

Is there any way to edit this?

**Comment by ThatXliner** [comment]
If you want it away, you can uncheck the box. Otherwise, you got the init script and your style sheet

**Reply by ghost** [reply]
If I wanted to have something like 'hello, world', instead of 'For help, please....', how could I do that?

**Comment by m1ga** [comment, marked as answer]
search the repo for the string and you'll find:
https://github.com/atom/atom/blob/10f7d454edb3d61e3c85f6be1c58122b656c295f/packages/welcome/lib/welcome-view.js#L97
there is the place to edit the welcome screen

**Reply by ghost** [reply]
https://discuss.atom.io/t/how-can-i-edit-core-packages/23016/5

This bridged the gap

**Comment by girouarddanielle** [comment]
Please take my email address off of this ,it will not let me unsubscribe
